http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -146,9 +147,11 @@ if [ "$IS_GCE_FORMATTED" == "false" ] ; then
   mkfs.ext4 -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/${DISK_DEVICE_ID}
 fi
 mount -o discard,defaults /dev/${DISK_DEVICE_ID} /work
-# Directory to store user installed packages so they persist
-mkdir -p /work/packages
-chmod a+rwx /work/packages
+# Directory to store user installed packages so they persist for non RStudio runtimes
+if [ -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
```

Comment:
Wonder if this block could be moved to the jupyter-specific setup.

Reply:
i like that approach better, will update